yarn.lock

```diff
@@ -3736,6 +3736,11 @@
   resolved "https://registry.yarnpkg.com/@types/d3-chord/-/d3-chord-2.0.3.tgz#3009b792b754da964d893b4269d1fe7757f21370"
```

Comment:
I think some of these dupe's might be caught if we use highest and fewest in the deps-test again.

Reply:
i'm okay with switching back